src/components/molecules/TitleInput.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+	<base-input
+		v-bind="{ ...$attrs, ...$props }"
+		:vmodel="vmodel"
+		classes="h1"
+		v-on="$listeners"
+	/>
+</template>
+<script>
+export default {
+	inheritAttrs: false,
+	model: {
+		prop: "vmodel",
+		event: "update:vmodel",
+	},
+	props: {
+		vmodel: {
+			type: [Array, String, Number, Boolean],
```

Comment:
```suggestion
			type: [String, Number],
```
I think Array and Boolean are not really usefull for a TitleInput because they can only appear for checkboxes and radio buttons

Reply:
yepp, was just copying